scripts/std-lib-regression.sh

```diff
@@ -2,14 +2,15 @@
 # Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
For git hygine, it would be better to put this as its own PR.

Reply:
Yeah, it's already in it's own PR (#593), but that PR was blocked by the github memory restriction. I could remove this commit from here, push this fix first and get back to the other PR later.